doc/Installation/CentOS-image.md

```diff
@@ -8,7 +8,13 @@ We have available for download a pre-built image based on CentOS 7. Details of t
 The image is built with VirtualBox, a vmdk is provided along with an ova which was exported using OFV 1.0 version. 
 These should be supported in VMWare Fusion, Workstation, Player and VirtualBox.
 
-Any issues with these images should be reported via [Github](https://github.com/librenms/librenms/issues) or our [Discord server](https://t.libren.ms/discord).
+* NOTE: It's highly recommended that you run updates by doing the follwoing. 
+``` cd /opt/librenms ``` 
+ then run daily.sh script 
```

Comment:
Single line comments go between single `

Multi line blocks go between ``` - you can also specify the type of block like ```bash

I'd just have the cd + ./daily.sh in a code block and leave out the `then run...`